test/parallel/test-net-listen-close-server-callback-is-not-function.js

```diff
@@ -0,0 +1,16 @@
+var common = require('../common');
+var assert = require('assert');
+var net = require('net');
+var gotError = false;
+
+var server = net.createServer(function(socket) {
+});
```

Comment:
If you don't expect the callback to get called, you can write this more succinctly as `var server = net.createServer(assert.fail)`.